core/src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -263,12 +263,12 @@ private[query] class EventsByTagPublisher(
     case ReplayDone(retrievedCount, deliveredCount, seqN, highest, mightBeMore) =>
       if (log.isDebugEnabled)
         log.debug(
-          s"${backtrackingLog}query chunk done for tag [{}], timBucket [{}], count [{}] of [{}]",
+          s"${backtrackingLog}query chunk done for tag [{}], timBucket [{}], count [{}] of [{}], mightBeMore [$mightBeMore]",
           tag, currTimeBucket, deliveredCount, retrievedCount
         )
       seqNumbers = seqN
       currOffset = highest
-      replayCountZero = retrievedCount == 0
+      replayCountZero = backtrackingMode == NoBacktracking && retrievedCount == 0
```

Comment:
`replayCountZero` is used in the end condition for `currentEventsByTag`.
I wonder if it makes sense to use the `LookingForDelayed` for the `currentEventsByTag` at all. The purpose is to find new persistence ids while the query is running and that is good for the "live" query but not sure if it makes any sense for the "current" query.